java/core/src/test/java/sleeper/core/partition/PartitionSubTreeFactoryTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022-2025 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.core.partition;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartitionSubTreeFactoryTest extends PartitionTreeTestBase {
```

Comment:
I'm finding it very difficult to verify these tests. Please can we use PartitionsBuilder to show the trees being used?

Reply:
I still can't see the tree in the tests. Take a look at my example test.